backend/app/mcp_server/tools/knowledge.py

```
@@ -0,0 +1,770 @@
+
+        db.commit()
+
+        return {
+            "success": True,
+            "document": {
+                "id": updated.id,
+                "name": updated.name,
+                "status": "pending",  # Re-indexing in progress
+                "mode": mode,
+            },
+        }
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)